stable/mariadb/README.md

```diff
@@ -45,21 +45,22 @@ The command removes all the Kubernetes components associated with the chart and
-| -----------------------     | ----------------------------------         | ---------------------------------------------------------- |
-| `image`                     | MariaDB image                              | `bitnami/mariadb:{VERSION}`                                |
-| `imagePullPolicy`           | Image pull policy.                         | `IfNotPresent`                                             |
-| `mariadbRootPassword`       | Password for the `root` user.              | `nil`                                                      |
-| `mariadbUser`               | Username of new user to create.            | `nil`                                                      |
-| `mariadbPassword`           | Password for the new user.                 | `nil`                                                      |
-| `mariadbDatabase`           | Name for new database to create.           | `nil`                                                      |
-| `persistence.enabled`       | Use a PVC to persist data                  | `true`                                                     |
-| `persistence.existingClaim` | Use an existing PVC                        | `nil`                                                      |
-| `persistence.storageClass`  | Storage class of backing PVC               | `nil` (uses alpha storage class annotation)                |
-| `persistence.accessMode`    | Use volume as ReadOnly or ReadWrite        | `ReadWriteOnce`                                            |
-| `persistence.size`          | Size of data volume                        | `8Gi`                                                      |
-| `resources`                 | CPU/Memory resource requests/limits        | Memory: `256Mi`, CPU: `250m`                               |
-| `config`                    | Multi-line string for my.cnf configuration | `nil`                                                      |
+|          Parameter          |                Description                 |                   Default                   |
+| --------------------------- | ------------------------------------------ | ------------------------------------------- |
+| `image`                     | MariaDB image                              | `bitnami/mariadb:{VERSION}`                 |
+| `imagePullPolicy`           | Image pull policy.                         | `IfNotPresent`                              |
+| `usePassword`               | Enable password authentication             | `true`                                      |
+| `mariadbRootPassword`       | Password for the `root` user.              | Randomly generated                          |
+| `mariadbUser`               | Username of new user to create.            | `nil`                                       |
+| `mariadbPassword`           | Password for the new user.                 | `nil`                                       |
```

Comment:
oops, forgot to update this section. @stela are you suggesting naming the secret key used here? Isn't that better described in NOTES.txt since you can actually refer to the real secret name?